packages/iTwinUI-react/package.json

```diff
@@ -33,7 +33,7 @@
     "build:watch": "yarn clean:build && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
     "clean:build": "yarn clean:coverage && rimraf esm && rimraf cjs",
     "clean:coverage": "rimraf coverage",
-    "clean": "yarn clean:coverage && yarn clean:build && rimraf node_modules",
+    "clean": "rimraf .turbo && yarn clean:coverage && yarn clean:build && rimraf node_modules",
```

Comment:
I'm pretty sure it doesn't, but does the order of deleting folders matter? I was thinking what if some cache is added deleting `.turbo` (when running the following command after removing `.turbo`)? E.g. if the commands following the `rimraf .turbo` fail for some reason, turbo might log it in the cache. But I don't think it would happen because you set the `cache` property in `turbo.json`'s `clean` command to false, but I just thought I'd confirm.

Reply:
I guess it would make sense in other scenario, but in here as I needed to disable the cache (otherwise `.turbo` folder gets locked and its impossible to remove it) it does not really matter. And in case it fails, logs in terminal with errors should be available